apps/nextjs-website/src/pages/[productSlug]/overview.tsx

```diff
@@ -1,6 +1,8 @@
 import { GetStaticPaths, GetStaticProps, GetStaticPropsResult } from 'next';
 import { Product } from '@/lib/types/product';
+
 import { getOverview, getOverviewPaths } from '@/lib/api';
+
 import Hero from '@pagopa/pagopa-editorial-components/dist/components/Hero';
```

Comment:
```suggestion
import { getOverview, getOverviewPaths } from '@/lib/api';
import Hero from '@pagopa/pagopa-editorial-components/dist/components/Hero';
```
Like the suggestion blank row sholud be removed in import section